keras_cv/models/backbones/detectron2/detectron2_backbone.py

```diff
@@ -0,0 +1,203 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+
+from keras_cv.backend import keras
+from keras_cv.layers.detectron2_layers import ViTDetPatchingAndEmbedding
+from keras_cv.layers.detectron2_layers import WindowedTransformerEncoder
+from keras_cv.models.backbones.backbone import Backbone
+from keras_cv.models.backbones.detectron2.detectron2_backbone_presets import (
+    backbone_presets,
+)
+from keras_cv.utils.python_utils import classproperty
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
```

Comment:
Sorry that this changed while this PR is in-flight, but if you sync to master this should now be

```
from keras_cv.api_export import keras_cv_export

@keras_cv_export("keras_cv.models.ViTDetBackbone")
```
(Same for all new public API symbols)

Reply:
Addressed in [ac7f30e](https://github.com/keras-team/keras-cv/pull/1987/commits/ac7f30ee2cb0ee7569244cf67dd18a50ef805752)